lib/tasks/maintenance.rake

```diff
@@ -4,25 +4,33 @@ namespace :maintenance do
   desc 'Enable the maintenance mode page ("reason", "allowed_paths", "allowed_ips" and "response_code" can be passed as environment variables)'
   rule /\Amaintenance:(.*:|)start\Z/ do |task|
     invoke_environment
+    if Gem.loaded_specs.has_key? 'redis' and Turnout.config.redis_url != nil
+      Turnout::RedisClient.client.set "turnout:maintenance", "default"
```

Comment:
Also nitpicky, but it would be nice to have `"turnout:maintenance"` as a constant somewhere.